README.md

```diff
@@ -3,29 +3,52 @@
 ### Prerequisites
  - [Docker](https://docs.docker.com/docker-for-mac) greater than or equal to `18.03.1-ce-mac64 (24245)`
 
+We recommend you install Docker from the link above, _not_ using homebrew.
 
 ### Setting up the project
 
-1. Copy the docker environment variables and fill in any missing secrets:
+1. Copy the docker environment variables and fill in any missing secrets from the TeachingJobs 1Password vault:
 
-```
+```bash
 $ cp docker-compose.env.sample docker-compose.env
 ```
 
 2. Build the docker container and set up the database
 
-```
+```bash
 bin/drebuild
 ```
 
 3. [Follow these instructions to configure HTTPS](config/localhost/https/README.md)
 
 4. Start the application
 
-```
+```bash
 bin/dstart
 ```
 
+## User accounts & data
+
+Before you can log in to the application locally you will need a __DfE Sign-in__ and an __invitation to join Teaching Jobs__. Talk to the team to get these set up.
+
+### Importing school data
+
+Populate your environment with real school data from the Edubase archive
```

Comment:
The service that we populate school information from is GIAS ([Get information about schools](http://get-information-schools.service.gov.uk/)) - edubase a legacy domain they use. It might be worth linking to this as this is also how we verify schools when issues come up e.g. https://get-information-schools.service.gov.uk/Establishments/Establishment/Details/140212